near-sdk-macros/src/lib.rs

```diff
@@ -11,6 +11,25 @@ use quote::quote;
 use syn::visit::Visit;
 use syn::{File, ItemEnum, ItemImpl, ItemStruct, ItemTrait};
 
+/// This macro is used on a struct and the function implementations
+/// to generate the necessary code to be a valid NEAR contract and expose
+/// the intended functions to be able to be called externally.
+///
+/// # Examples
+///
+/// ```ignore
```

Comment:
```suggestion
/// ```no_run
```

Why not compile it to make sure docs stay up to date?

Reply:
I think this would have the same problem, the test would fail because of dependency cycle.